homeassistant/components/upb/const.py

```diff
@@ -13,7 +13,7 @@
 ATTR_COMMAND = "command"
 ATTR_RATE = "rate"
 CONF_NETWORK = "network"
-EVENT_UPB_LINK_CHANGED = "upb.link_changed"
+EVENT_UPB_LINK_CHANGED = "upb.scene_changed"
```

Comment:
We should change the constant name too, probably. That's standard to have the constant and string match.